--- origsrc/gd-2.0.36RC1/configure.ac	2007-11-27 09:30:34.000000000 +0100
+++ src/gd-2.0.36RC1/configure.ac	2014-12-08 14:14:43.085228700 +0100
@@ -351,28 +351,17 @@ AC_ARG_WITH(png,dnl
 
 if test "$withval" != no; then
   if test -d "$withval"; then
-    if test -e "$withval/bin/libpng12-config"; then
-      LIBPNG12_CONFIG="$withval/bin/libpng12-config"
-    fi
     if test -e "$withval/bin/libpng-config"; then
       LIBPNG_CONFIG="$withval/bin/libpng-config"
     fi
   else
-      AC_PATH_PROG([LIBPNG12_CONFIG], [libpng12-config])
       AC_PATH_PROG([LIBPNG_CONFIG], [libpng-config])
   fi
 
-  if test -n "$LIBPNG12_CONFIG"; then
-    libpng_CPPFLAGS=`$LIBPNG12_CONFIG --cflags`
-    # should be --ldopts, but it's currently broken
-    libpng_LDFLAGS=`$LIBPNG12_CONFIG --ldflags`
-    libpng_LDFLAGS=`echo " $libpng_LDFLAGS" | sed 's/ -l[[^ ]][[^ ]]*//g'`
-  elif test -n "$LIBPNG_CONFIG"; then
-    libpng_CPPFLAGS=`$LIBPNG_CONFIG --cflags`
-    # should be --ldopts, but it's currently broken
-    libpng_LDFLAGS=`$LIBPNG_CONFIG --ldflags`
-    libpng_LDFLAGS=`echo " $libpng_LDFLAGS" | sed 's/ -l[[^ ]][[^ ]]*//g'`
-  fi
+  libpng_CPPFLAGS=`$LIBPNG_CONFIG --cflags`
+  # should be --ldopts, but it's currently broken
+  libpng_LDFLAGS=`$LIBPNG_CONFIG --ldflags`
+  libpng_LDFLAGS=`echo " $libpng_LDFLAGS" | sed 's/ -l[[^ ]][[^ ]]*//g'`
 
   _cppflags="$CPPFLAGS"
   CPPFLAGS="$CPPFLAGS $libpng_CPPFLAGS"
@@ -385,14 +374,7 @@ if test "$withval" != no; then
   _ldflags="$LDFLAGS"
   LDFLAGS="$libpng_LDFLAGS $LDFLAGS"
 
-  if test -n "$LIBPNG12_CONFIG"; then
-    AC_CHECK_LIB(png12,png_create_read_struct,
-      [LIBS="-lpng12 $LIBS"
-       FEATURES="GD_PNG $FEATURES"
-       AC_DEFINE(HAVE_LIBPNG, 1, [ Define if you have the png library. ])],
-      [LDFLAGS="$_ldflags"  
-       AC_MSG_WARN([libpng is required - see http://www.libpng.org/pub/png/])])
-  elif test -n "$LIBPNG_CONFIG"; then
+  if test -n "$LIBPNG_CONFIG"; then
     AC_CHECK_LIB(png,png_create_read_struct,
       [LIBS="-lpng $LIBS" 
        FEATURES="GD_PNG $FEATURES"
